lib/redis/connection/command_helper.rb

```diff
@@ -1,8 +1,10 @@
+# frozen_string_literal: true
 class Redis
   module Connection
     module CommandHelper
 
-      COMMAND_DELIMITER = "\r\n"
+      COMMAND_DELIMITER = "\r\n".freeze
```

Comment:
You don't need these `.freeze` if you also put the `# frozen_string_literal: true` magic comment.